### PR TITLE
Update map styling

### LINKED ISSUE #96: Update Map styling
- Make Markers red (specifically `#FC323E`) via `color` property in `Marker`

> Fun fact: hex code colours are just Red, Green, and Blue values in the format `#RRGGBB`, where each value is a hexadecimal (base-16) number from 1 to F (16)

- Use our Mapbox styling (`mapbox://styles/mumap/ckhb8l6je01ki1ao4tabh8e2q`) rather than default `streets-v11` style
- Replace Map.css with React's `useStyles` for consistency with other components
- Maybe add [Navigation Controls](https://docs.mapbox.com/mapbox-gl-js/example/navigation/) while you're at it! These aren't in Figma but will be useful for now and are easy to remove if need be

### PR DESCRIPTION
**Pull request by tedspare**
# Summary

- Replaced default Mapbox styles with MU's map style
- Added Mapbox NavigationControls (see below)
- Updated marker colour to reflect Figma

## Test Plan

Ran in the browser

## Related Issues

Closes #96 
Did it for the _momentum_ 🏃‍♂️

### CODE REVIEW

File: frontend/src/Map/Map.tsx [hunk 1]
```diff
@@ -48,7 +63,7 @@ function Map({ mapContainer, murals }: IMapProps) {
     });
 
     murals.forEach((mural: any) => {
-      new mapboxgl.Marker()
+      new mapboxgl.Marker({ color: "#FC323E" })
```
Comment (JDziewonski98) [inline frontend/src/Map/Map.tsx:66]: Maybe parametrize this or put in constants? Not big deal

**Review by JDziewonski98 — APPROVED** [review]
Everything makes sense, LGTM

File: frontend/src/Map/Map.tsx (hunk 1 above)
Comment (tedspare) [reply frontend/src/Map/Map.tsx:66]: I realized this issue might get closed by Blanche's next PR so I'll wait a bit to merge

**Review by Gabriel-Paquette — APPROVED** [review]
Everything LGTM

**Comment by tedspare** [comment]
Closed by #119

**Pull request closed without merging**